src/util/apis/generated/operations/buildOperations.js

```diff
@@ -860,7 +860,7 @@ BuildOperations.prototype.getCommits = function (shaCollection, ownerName, appNa
   // Construct URL
   var baseUrl = this.client.baseUri;
   var requestUrl = baseUrl + (baseUrl.endsWith('/') ? '' : '/') + 'v0.1/apps/{owner_name}/{app_name}/commits/batch/{sha_collection}';
-  requestUrl = requestUrl.replace('{sha_collection}', encodeURIComponent(shaCollection));
```

Comment:
I took a look, there's a switch in autorest we can use to turn off the url encoding instead of having to manually patch the code. I'll add a thing to the swagger fixup script to add this manually while I yell at build to update their swagger.